src/cunumeric/matrix/batched_cholesky_template.inl

```diff
@@ -57,7 +57,7 @@ struct _cholesky_supported {
 
 template <VariantKind KIND>
 struct BatchedCholeskyImpl {
-  template <Type::Code CODE, int DIM>
+  template <Type::Code CODE, int32_t DIM, std::enable_if_t<2 < DIM>* = nullptr>
```

Comment:
nit: `DIM > 2` is more readable and matches the other enable_if

Reply:
Yup, I convinced the compiler to accept it by adding parens dd5e01d